store/index.js

```diff
@@ -62,7 +64,7 @@ export const store = new Vuex.Store({
     de,
     en,
     es,
-    nl,
+    it,
```

Comment:
I removed `nl` because it is also defined at line 69.

Reply:
Thanks!